RX interruption catched
TASK_ALLOCATION_SEND
Got slot: 0 
Task total size (txt+bss): 3124 
Task addr: 2147522956 
Task slot: 0 
Task slot TaskAddr: 2147522956 
RX interruption catched
TASK_ALLOCATION_FINISHED
SendingQueue_front: 0 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um TASK_START!
RX interruption catched
Chegou um TASK_RUN!
Starting Task 2  from app 1 
taskSlot run: 0 
SendingQueue_front: 1 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
Task P2 INIT
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 3 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 5 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 6 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 7 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 3 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 5 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 2037186 
TotalEnergy: 21506416 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 961621 
TotalEnergy: 42424629 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 999900 
TotalEnergy: 43931719 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1000001 
TotalEnergy: 44164796 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1000001 
TotalEnergy: 44167159 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1000073 
TotalEnergy: 44166265 
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
Esperando Mensagem!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1045653 
TotalEnergy: 46163900 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 954085 
TotalEnergy: 42360957 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1000001 
TotalEnergy: 44178253 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 999900 
TotalEnergy: 43936412 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1000001 
TotalEnergy: 43938428 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1000001 
TotalEnergy: 44175695 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1003334 
TotalEnergy: 44309872 
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send failed - NI_TX occupied!
Esperando Mensagem!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1041411 
TotalEnergy: 45977306 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 955056 
TotalEnergy: 42398205 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1000102 
TotalEnergy: 44180151 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 999698 
TotalEnergy: 44162388 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1000001 
TotalEnergy: 44156209 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1000059 
TotalEnergy: 44177919 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 999900 
TotalEnergy: 43937717 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3953125 
total instructions~~~> 999938 
TotalEnergy: 43473653 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1000102 
TotalEnergy: 43479467 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3953125 
total instructions~~~> 999900 
TotalEnergy: 43471621 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1000001 
TotalEnergy: 43475473 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1000001 
TotalEnergy: 43475455 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3953125 
total instructions~~~> 999972 
TotalEnergy: 43473913 
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3953125 
total instructions~~~> 999958 
TotalEnergy: 43473254 
TX interruption catched
API_Try2Send failed - empty SendQueue!
